src/main/java/ch/sbb/polarion/extension/pdf/exporter/util/configuration/ConfigurationStatusUtils.java

```diff
@@ -82,6 +83,21 @@ public class ConfigurationStatusUtils {
         }
     }
 
+    public static @NotNull ConfigurationStatus getWeasyPrintServiceStatus() {
+        try {
+            HtmlToPdfConverter htmlToPdfConverter = new HtmlToPdfConverter();
+            WeasyPrintConverter weasyPrintConverter = htmlToPdfConverter.getWeasyPrintConverter();
+            if (weasyPrintConverter instanceof WeasyPrintServiceConnector) {
+                ModuleDescriptor.Version weasyPrintServiceVersion = ((WeasyPrintServiceConnector) weasyPrintConverter).getWeasyPrintServiceVersion();
```

Comment:
if the version is not provided, so I assume it will be null, we need to show `unknown`

Reply:
but here seems to be NPE

Reply:
Fixed